### ISSUE TITLE
DCGANs Tutorial's BatchNormalization is maybe something wrong

### ISSUE THREAD

**Issue by MokkeMeguru** [tensorflow/tensorflow#35472]
## URL(s) with the issue:
https://www.tensorflow.org/tutorials/generative/dcgan

## Description of issue (what needs changing):

https://colab.research.google.com/gist/MokkeMeguru/614e16d83d16f1eb70b5f3b73c7d070b/batchnormalization_debug.ipynb

In you tutorial, BatchNormalization will be Actnormalization in Glow(https://arxiv.org/abs/1807.03039)

### Clear description

We need Correct BathNormalization

### Usage example

We should input the shape when   BatchNormalization is initialized

**Comment by MokkeMeguru** [comment]
Sorry, my misunderstanding about batch normalization.
